docs/getting-started/quickstart.md

```diff
@@ -31,6 +31,13 @@ Next, log into your instance of Infra to setup your admin account:
 infra login localhost
 ```
 
+You may be prompted to verify the fingerprint of the server's TLS certificate. The
```

Comment:
Move this to the callout below?

Reply:
Ya, that could work. Do you think we should have two different callouts together in the same block, or have two separate callout blocks?

Reply:
One block may be better to avoid "callout exhaustion"

Reply:
Cool, let's try that. It might be less obvious that there are two different callouts in that block, but easy to change later if we need to.